tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -82,6 +82,19 @@ public DotNetCoreDynamicInstallTest(ITestOutputHelper output) : base(output)
                 appName, runtimeVersion, _imageHelper.GetCliImage(ImageTestHelperConstants.CliBusterRepository));
         }
 
+        [Theory, Trait("category", "cli-bullseye")]
+       // [InlineData(NetCoreApp21WebApp, "2.1")]
+        [InlineData(NetCoreApp31MvcApp, "3.1")]
+       // [InlineData(NetCoreApp50MvcApp, "5.0")]
+        [InlineData(NetCore7PreviewMvcApp, "7.0")]
+        public void BuildsApplication_ByDynamicallyInstallingSDKs_CliBullseye(
```

Comment:
Seems like dotnet 2.1 and dotnet 5.0 are not among the supported versions? Happy to remove these instead of commenting after confirming.

Reply:
.NET Core 3.1 is EOL as well, so we may be able to scrap that as well? I'd defer to @daniv-msft

Reply:
+1 on deleting the obsolete versions at this point. And another +1 to include 3.1 in the versions we delete as well, as we no longer support it.

Reply:
Removed these for this test.